concourse/scripts/unbind_job.rb

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env ruby
+require 'yaml'
+pipe = YAML.load(STDIN)
+
+def clean_passed(obj)
```

Comment:
Can we call this `remove_passed` or `delete_passed` to give a better indication of what it's doing?